Milyli.ScriptRunner.Services.Contracts/V1/Enumerations/RunStatus.cs

```diff
@@ -0,0 +1,12 @@
+﻿namespace Milyli.ScriptRunner.Services.Contracts.V1.Enumerations
+{
+	/// <summary>
+	/// Describes status of a Script Run.
+	/// </summary>
+	public enum RunStatus
+	{
+		Idle = 0,
```

Comment:
The WeeklySchedule enum is pretty self-explanatory but this one could use xmldocs on the individual values.

- [ ] XMLDoc enum values